frontend/src/app/modules/time-series/services/line-chart.service.ts

```diff
@@ -166,14 +177,16 @@ export class LineChartService {
   private getYScale(data: TimeSeries[]): D3ScaleLinear<number, number> {
     return d3ScaleLinear()              // Linear scale for the numbers on the Y-Axis
              .range([this._height, 0])  // Display the Y-Axis over the complete height - origin is top left corner, so height comes first
-             .domain([                  // min and max values from the data given
-               0, d3Max(data, (dataItem: TimeSeries) => {
-                 return d3Max(dataItem.values, (point: TimeSeriesPoint) => {
-                   return point.value;
-                 });
-               })
-             ])
-             .nice();
+             .domain([0, this.getMaxValue(data)])
+             //.nice();
```

Comment:
Do we not want to have `nice` padding in the chart?

Reply:
Uncommented